website/views/social/achievement.jade

```diff
@@ -0,0 +1,9 @@
+extends ./layout
+
+block extraHead
+  - var socialPage = 'achievement'
+  - var socialTitle = 'Earned an Achievement!'
+  - var socialDescription = 'I earned a new Achievement on Habitica for working to improve my life!'
```

Comment:
Ah, can we? I'm fuzzy on how that works. This is something that'd get crawled and displayed when somebody examines a Tweet (or FB share, etc.) on a timeline. Would the meta tags get translated then?